index.bs

```diff
@@ -1915,8 +1971,6 @@ To <dfn>parse trigger specs</dfn> given a [=map=] |map|, a [=moment=]
         1. Set |i| to |i| + 1.
 1. Return |specs|.
 
-Issue: Parse "`summary_window_operator`" and "`summary_buckets`" fields.
```

Comment:
Let's keep this issue but rephrase it as something like `Issue: Invoke [=parse summary buckets=] from this algorithm.`